ocs_ci/ocs/resources/objectbucket.py

```diff
@@ -225,7 +226,11 @@ def delete(self, verify=True):
             logger.warning(f"{self.name} deletion timed out. Verifying deletion.")
             verify = True
         if verify:
-            self.verify_deletion()
+            # Increae the timeout to 15 minutes if the test is tier4
```

Comment:
"Increae" small typo. Fix/not - up to you

Reply:
Thanks for pointing it out - fixed